Source/Private/ASDisplayNodeInternal.h

```diff
@@ -326,6 +335,9 @@ FOUNDATION_EXPORT NSString * const ASRenderingEngineDidDisplayNodesScheduledBefo
  */
 - (void)nodeViewDidAddGestureRecognizer;
 
+// Recalculates fallbackSafeAreaInsets for the subnodes
+- (void)_fallbackUpdateSafeAreaOnChildren;
```

Comment:
Nit: May not needed because `_fallbackUpdateSafeAreaOnChildren` is implemented and called in `ASDisplayNode.mm` only.

Reply:
This is also used by `-[ASDisplayNode safeAreaDidChange]`, which is implemented in `ASDisplayNode+UIViewBridge.mm`.